### ISSUE TITLE
Tag on push to master

### ISSUE THREAD

**Issue by joaomilho** [klarna/ui-react-components#35]
Instead of tagging manually, we should tag on the same process that publishes the package, on a master push.

**Comment by xaviervia** [comment]
True, but we need to be able to parametrize what the version bump is going to be to do that. Should we do that reading from the merge commit message or something like that? Maybe we should add a step in codeship that captures `[patch]`, `[minor]`, `[major]` in the commit messages and runs the npm version accordingly.

Actually when I think about it, if we also take the opportunity to write the CHANGELOG entry in the merge commit message, we can automate the whole bump and publish process including CHANGELOG update and release metadata as requested in https://github.com/klarna/ui-css-components/issues/31

**Comment by JoakimLofgren** [comment]
Yeah, using the GitHub releases feature would be nice
